uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -1185,8 +1185,8 @@ public function testGetAndGetFullReturnAllLeaveRequestsWhenTheExpiredParamIsNotP
       ['period_start_date' => '2016-01-01']
     );
 
-    // This request has 3 days expired, but will be included on
-    // the response anyway, since the "expired" flag is not set
+    // This request has 3 days expired, but will not included on
```

Comment:
Guess a `be` is missing in the comment. 
`but will not be included on`

Kindly check the  `set/false` on the next line as well.